projects/storefrontlib/src/lib/checkout/index.ts

```diff
@@ -1,11 +1,32 @@
 export * from './checkout.module';
-export * from './components/shipping-address/shipping-address.module';
+export * from './components/checkout-orchestrator/checkout-orchestrator.component';
```

Comment:
Lets export service and model also. Also move components exports to other `components` directory barrel and export only this barrel